pennylane_lightning/lightning_qubit.py

```diff
@@ -32,12 +33,25 @@
 from ._version import __version__
 
 try:
-    from .lightning_qubit_ops import (
-        apply,
-        StateVectorC64,
-        StateVectorC128,
-        AdjointJacobianC128,
-    )
+    print(os.path.dirname(os.path.abspath(__file__)))
+    if platform.system() == "Windows" and sys.version_info[:2] >= (3, 8):
+        # Add the current directory to DLL path.
+        # See https://docs.python.org/3/whatsnew/3.8.html#bpo-36085-whatsnew
```

Comment:
Will this have any problem with Python 3.7 support on Windows?

Reply:
Python version below 3.8 follows the legacy (which means the `else` part will be executed without a problem). `cibuildwheel` indeed tests Windows+Python3.7 which do not report any problem.